receiver/mongodbreceiver/scraper.go

```diff
@@ -191,3 +190,7 @@ func (s *mongodbScraper) recordAdminStats(now pcommon.Timestamp, document bson.M
 func (s *mongodbScraper) recordIndexStats(now pcommon.Timestamp, indexStats []bson.M, databaseName string, collectionName string, errs *scrapererror.ScrapeErrors) {
 	s.recordIndexAccess(now, indexStats, databaseName, collectionName, errs)
 }
+
+func unknownVersion() *version.Version {
+	return version.Must(version.NewVersion("0.0"))
+}
```

Comment:
We can instantiate it once like this.
```suggestion
var unknownVersion = func() *version.Version {
	return version.Must(version.NewVersion("0.0"))
}()
```

Reply:
done